src/celestia/qt/qtappwin.cpp

```diff
@@ -425,7 +425,7 @@ void CelestiaAppWindow::init(const QString& qConfigFileName,
 
     connect(fullScreenAction, SIGNAL(triggered()), this, SLOT(slotToggleFullScreen()));
     viewMenu->addAction(fullScreenAction);
-
+    
```

Comment:
extra spaces. you can a git hook to prevent commits with extra spaces. in `.git/hooks/` you have a `pre-commit.sample`, just remove `.sample`

Reply:
Thanks. Recently my text editor lost ability to highlight extra spaces, not sure why.